nbs/core.ipynb

```diff
@@ -94,7 +94,8 @@
     "    MLPMultivariate, iTransformer,\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.    pred_airline2 = pred_dataframe[pred_dataframe['id'] == 'Airline2'].reset_index(drop=True)
</pre><p>I don't think the reset index is needed here, we set the env variable to return the id as a column above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
yes removing it seems to still work well